index.js

```diff
@@ -66,24 +66,29 @@ function renderElements(timestamp) {
 }
 
 
+/**
+ * @typedef {new(...args: any[]) => any} Newable         Type alias for a really generic class constructor
+ * @typedef {Newable}                    SvelteComponent Svelte component class constructor (basically a "newable" object)
```

Comment:
Consider rename, e.g. maybe `ComponentType` (https://svelte.dev/docs/typescript#types-componenttype) or make own like `ComponentConstructor` for clarity, since in Svelte, `SvelteComponent` actually exists and is an instance type… I think.

If `ComponentType` is used though, it should probably be actually imported.